spk/salt-minion/Makefile

```diff
@@ -0,0 +1,57 @@
+SPK_NAME = salt-minion
+SPK_VERS = 3001.1
+SPK_REV = 1
+SPK_ICON = src/salt-minion.png
+
+PIP = $(WORK_DIR)/../../../native/python3/work-native/install/usr/local/bin/pip
+BUILD_DEPENDS = cross/python3
+DEPENDS = cross/busybox
+SPK_DEPENDS = "python3>=3.7"
+
+# Requirements file generation
+# /usr/local/python3/bin/python3 -mvirtualenv --python=python3  salt-env
+# virtualenv --python=python3  salt-env
+# source salt-env/bin/activate
+# pip install salt
+# pip freeze > requirements.txt
+WHEELS = src/requirements.txt
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
+RELOAD_UI = yes
+DISPLAY_NAME = Salt Minion
+CHANGELOG = "Update to 3001.1"
+
+HOMEPAGE   = https://www.saltstack.com/
+LICENSE    = Apache
+
+POST_STRIP_TARGET = salt-minion_extra_install
+
+# Service configuration
+CONF_DIR = src/conf
+SERVICE_USER = auto
+SERVICE_SETUP = src/service-setup.sh
+STARTABLE = yes
+
+# Busybox config
+BUSYBOX_CONFIG = salt-minion
+ENV += BUSYBOX_CONFIG="$(BUSYBOX_CONFIG)"
```

Comment:
busybox is already available in Python 3 package and may be re-used. Have you only embedded it for need of "patch"? If so, I prefer to enable "patch" in Python 3 busybox to avoid to rebuild/embed it again in salt-minion

Reply:
Initially I added busybox just to have the "patch" command, but then I added some system utils that may be needed by salt recipes. If you prefer a "cleaner" build I can do that: Add the "patch" applet to Python3's busybox, and create e new package if I need aditional tools for my salt recipes

Reply:
Ok, I already restored the full privileges file